ftl/BUILD

```diff
@@ -0,0 +1,24 @@
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@io_bazel_rules_docker//docker/contrib/python:image.bzl",
+    "py_image",
+)
+load(
+    "@io_bazel_rules_docker//docker:docker.bzl",
+    "docker_push",
+    "docker_build",
+)
+load("@subpar//:subpar.bzl", "par_binary")
```

Comment:
You don't need any of these `load` statements.

Reply:
Cleaned.